sp7_db_setup_check.sh

```
@@ -138,125 +177,68 @@ else
-      echo "--------------"
-      echo "GRANT ALL PRIVILEGES ON \`${DB_NAME}\`.* TO '${MIGRATOR_NAME}'@'${h}'"
-      echo "--------------"
-      exit 1
-    fi
-  done
-
-  if ! mariadb -h "$DB_HOST" -P "$DB_PORT" \
-        -u "$MASTER_USER_NAME" --password="$MASTER_USER_PASSWORD" \
-        -e "FLUSH PRIVILEGES;"; then
-    echo "Error: Failed to FLUSH PRIVILEGES for migrator user."
+if [[ "$NEW_MIGRATOR_USER_CREATED" -eq 1 ]]; then
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this then clause.

(SC1009)